test/e2e/framework.go

```diff
@@ -1152,3 +1177,109 @@ func (data *TestData) mutateAntreaConfigMap(mutatingFunc func(data map[string]st
 	}
 	return nil
 }
+
+// gracefulExitAntreaController copies the Antrea controller binary coverage data file out before terminating the Pod
+func (data *TestData) gracefulExitAntreaController(covDir string) error {
+	antreaController, err := data.getAntreaController()
+	if err != nil {
+		return fmt.Errorf("error when getting antrea-controller Pod: %v", err)
+	}
+	podName := antreaController.Name
+	cmds := []string{"pgrep", "-o", antreaControllerCovBinary, "-P", "1"}
+	stdout, stderr, err := data.runCommandFromPod(antreaNamespace, podName, "antrea-controller", cmds)
+	if err != nil {
+		return fmt.Errorf("error when getting pid of antreaContollerCovBinary: <%v>, err: <%v>", stderr, err)
```

Comment:
"antreaContollerCovBinary" is not ideal in a log message

if you set `antreaContollerCovBinary` to `"antrea-controller-coverage"` as suggested above, you can replace this error message with:
```go
fmt.Errorf("error when getting pid of '%s': <%v>, err: <%v>", antreaContollerCovBinary, stderr, err)
```

same for other log messages, including for the agent

Reply:
right, this makes more sense. Thanks Antonin.